### PR TITLE
Fix example

### PR DESCRIPTION
**Pull request by LAMike310**
Replaced satoshi@getalby.com with alby@getalby.com in the README.md example

### CODE REVIEW

**Comment by LAMike310** [comment]
I was getting an error trying to run the example code, but alby@getalby.com works fine

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The `LightningAddress` class provides helpers to work with lightning addresses
 ```js
 import { LightningAddress } from "alby-tools";
 
-const ln = new LightningAddress("satoshi@getalby.com");
+const ln = new LightningAddress("alby@getalby.com");
```
Comment (bumi) [inline README.md:28]:
```suggestion
const ln = new LightningAddress("hello@getalby.com");
```

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ console.log(ln.keysendData);
...
```
Comment (bumi) [inline README.md:45]:
```suggestion
const ln = new LightningAddress("hello@getalby.com");
```

File: README.md [hunk 3]
```diff
@@ -57,7 +57,10 @@ console.log(invoice.paymentHash); // print the payment hash
 
 ```js
 import { LightningAddress } from "alby-tools";
-const ln = new LightningAddress("satoshi@getalby.com");
+const ln = new LightningAddress("
+
+
+");
```
Comment (bumi) [inline README.md:63]:
```suggestion
const ln = new LightningAddress("hello@getalby.com");
```

**Pull request merged**